client/src/types/phaser.type.ts

```diff
@@ -0,0 +1,70 @@
+export const Direction = {
+    LEFT: "left",
+    RIGHT: "right",
+    UP: "up",
+    DOWN: "down",
+} as const;
+
+export type IDirection = (typeof Direction)[keyof typeof Direction];
+
+export interface PlayerInfo {
```

Comment:
```suggestion
export const DIRECTION = {
    left: "left",
    right: "right",
    up: "up",
    down: "down",
} as const;

export type Direction = (typeof DIRECTION)[keyof typeof DIRECTION];

export interface IPlayerInfo {
```
이제 컨벤션이 눈에 들어왔어요..ㅋㅋㅋ이까지만 변경했어요
https://www.notion.so/CLIENT-f82a99c009e84237b82400cf1410ccfe?pvs=4#96c651e28fc34bd0ae78e35250b8c6e0

`type`은 따로 정한 적이 없어서 곧 논의해봅시다